requirements.txt

```diff
@@ -0,0 +1,30 @@
+aspy.yaml==1.3.0
```

Comment:
should requirements be hard locked to versions or allow for minor version increases?

Reply:
The requirements are hard locked because that's the standard output from `pip freeze`, allowing minor version increases could be good, but it's also unlikely that hard locking the versions will cause any problems for a project this size